### PR TITLE
Increase scroll duration

### PR DESCRIPTION
**Pull request by prashant-saini**
Added Extend Scroll duration method for the currently running animation.
I copied the code of Android ViewPager class and updated it.

### CODE REVIEW

**Review by saxenajabong — APPROVED** [review]
Looks Good To Me (LGTM).

File: android/build.gradle [hunk 1]
```diff
@@ -9,23 +9,25 @@ buildscript {
...
+    buildToolsVersion versions.buildTools
 
     defaultConfig {
-        minSdkVersion getExtOrIntegerDefault('minSdkVersion')
-        targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.targetSdk
+    }
+
+    buildTypes {
```
Comment (anuragdadheech) [inline android/build.gradle]: We don't need to do this anymore. Check with Ankit Gupta once.

Comment (prashant-saini) [reply android/build.gradle]: Discussed it with Ankit. Removed buildtypes block.

**Review by anuragdadheech — COMMENTED** [review]
Looks good

**Pull request merged**